shacl12-ui/index.html

```diff
@@ -1124,15 +1124,15 @@ <h4>shui:TextAreaWithLangEditor</h4>
           <h4>shui:TextFieldEditor</h4>
           <p>
             <b>Score:</b>
```

Comment:
I'll leave the rest to you.
```suggestion
            <b>Score:</b><br>
```

Reply:
The lines below *Score:* are now a list, which enforces line breaks.